tests/jeos/firstrun.pm

```diff
@@ -93,7 +93,7 @@ sub verify_norepos {
 sub verify_bsc {
     if (is_qemu && is_x86_64 && script_run("rpm -q qemu-guest-agent") != 0) {
         # Included in SLE-15-SP2+, TW and Leap
-        die("bsc#1207135 - Missing qemu-guest-agent from virtual images") unless is_sle('<15-SP2');
+        die("bsc#1207135 - Missing qemu-guest-agent from virtual images") unless is_sle('<15-SP2') || is_bootloader_sdboot;
```

Comment:
How is this related to systemd boot? If the package is in other TW images, aren't we covering a bug?

Reply:
Thanks for pointing that out. I wasn't sure about this one, but the fact is that qemu-guest-agent does not come installed in the JeOS-for-kvm-and-xen-sdboot image. The sub verify_norepos is only ran if is_jeos, so I thought adding that unless wouldn't be harmful. Please advise :)

Reply:
I would go for a bug report as the qemu-ga is recommended to be used in any qemu VM.

Reply:
Will do! As for this PR, I removed the change in https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/20804/commits/fa5412fa1d25cdf3f42de73cabf567502a857d34

Reply:
https://bugzilla.opensuse.org/show_bug.cgi?id=1234610